html/modules/custom/bc_dc/bc_dc.module

```diff
@@ -276,3 +276,22 @@ function bc_dc_cron(): void {
   $reviewReminder = \Drupal::service('bc_dc.review_reminder');
   $reviewReminder->sendRemindersToAllUsers();
 }
+
+/**
+ * Implements hook_form_alter.
+ */
+function bc_dc_form_alter(&$form, \Drupal\Core\Form\FormStateInterface $form_state, $form_id) {
+  if ($form['form_id']['#id'] === 'edit-bc-dc-add-columns-form') {
+    $nid = $form_state->getStorage('nid');
+    $paragraph = \Drupal::entityTypeManager()->getStorage('node')->load($nid['nid']);
+    $paragraph_field_items = $paragraph->get('field_columns')->referencedEntities();
+    if ($paragraph_field_items) {
+      $form['data_dictionary'] = [
+        '#markup' =>
```

Comment:
This should use `item_list`.

Reply:
@lkmorlan which line should be 'item_list'?

Reply:
I mean to theme it as `'#type' => 'item_list'` instead of using `#markup`.

Reply:
fixed